src/index.ts

```diff
@@ -1,7 +1,8 @@
 import * as puppeteer from 'puppeteer';
-import downloader from './metamaskDownloader';
 
-const timeout = seconds => new Promise(resolve => setTimeout(resolve, seconds * 1000))
+import _default from '../dist/metamaskDownloader';
```

Comment:
remove unused import

Reply:
resolved in ci/cd PR gonna close this and merge all together